src/test/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestExtractFileValidationServiceImplTest.java

```diff
@@ -16,41 +17,72 @@
-        List<ConcurRequestExtractFile> testFiles = ConcurRequestExtractFileFixture.GOOD_FILE.createConcurRequestExtractFiles();
-        assertTrue("Expected Result: Header amount SHOULD match sum of row amounts from file.", concurRequestExtractFileValidationService.requestExtractHeaderRowValidatesToFileContents(testFiles));
+        ConcurRequestExtractFile testFile = ConcurRequestExtractFileFixture.GOOD_FILE.createConcurRequestExtractFile();
+        assertTrue("Expected Result: Header amount SHOULD match sum of row amounts from file.", concurRequestExtractFileValidationService.requestExtractHeaderRowValidatesToFileContents(testFile));
     }
     
     @Test
     public void testHeaderAmountDoesNotMatch() {
-        List<ConcurRequestExtractFile> testFiles = ConcurRequestExtractFileFixture.BAD_REQUEST_AMOUNT_FILE.createConcurRequestExtractFiles();
-        assertFalse("Expected Result: Header amount should NOT match sum of row amounts from file.", concurRequestExtractFileValidationService.requestExtractHeaderRowValidatesToFileContents(testFiles));
+        ConcurRequestExtractFile testFile = ConcurRequestExtractFileFixture.BAD_REQUEST_AMOUNT_FILE.createConcurRequestExtractFile();
+        assertFalse("Expected Result: Header amount should NOT match sum of row amounts from file.", concurRequestExtractFileValidationService.requestExtractHeaderRowValidatesToFileContents(testFile));
     }
 
     @Test
     public void testHeaderRowCountDoesNotMatch() {
-        List<ConcurRequestExtractFile> testFiles = ConcurRequestExtractFileFixture.BAD_FILE_COUNT_FILE.createConcurRequestExtractFiles();
-        assertFalse("Expected Result: Header row count should NOT match file row count.", concurRequestExtractFileValidationService.requestExtractHeaderRowValidatesToFileContents(testFiles));
+        ConcurRequestExtractFile testFile = ConcurRequestExtractFileFixture.BAD_FILE_COUNT_FILE.createConcurRequestExtractFile();
+        assertFalse("Expected Result: Header row count should NOT match file row count.", concurRequestExtractFileValidationService.requestExtractHeaderRowValidatesToFileContents(testFile));
+    }
+
+    @Test
+    public void testFileContainsBadEmployeeGroupId() {
+        ConcurRequestExtractFile testFile = ConcurRequestExtractFileFixture.BAD_EMPLOYEE_GROUP_ID_FILE.createConcurRequestExtractFile();
+        assertFalse("Expected Result: Request Detail row contains BAD Employee Group Id.", concurRequestExtractFileValidationService.requestExtractHeaderRowValidatesToFileContents(testFile));
+    }
+
+    private class TestableConcurBatchUtilityServiceImpl extends ConcurBatchUtilityServiceImpl {
+        @Override
+        public String getConcurParamterValue(String parameterName) {
+            String parameterValue = concurParameterConstantsFixture.getValueForConcurParamter(parameterName);
+            if (StringUtils.isEmpty(parameterValue)) {
+                return parameterName;
```

Comment:
If the parameter does not exist, I would suggest returning null instead of returning the parameter name.